site/content/en/docs/drivers/qemu.md

```diff
@@ -13,6 +13,20 @@ The `qemu` driver users QEMU (system) for VM creation.
 
 <https://www.qemu.org/>
 
+## Usage
+
+To start minikube with the qemu driver:
+
+```shell
+minikube start --driver=qemu
+```
+
+To make qemu the default driver: 
```

Comment:
lets not advertise making it default for all clusters (since this is an expermintal)

Reply:
Sure, we can remove this section on making qemu default.